src/index.js

```diff
@@ -0,0 +1,54 @@
+const pretty = require('pino-pretty')
+
+const LEVEL_TO_STRING = {
+  60: 'fatal',
+  50: 'error',
+  40: 'warn',
+  30: 'info',
+  20: 'debug',
+  10: 'trace'
+}
+
+const formatTime = (timestamp) => {
+  const dateObject = new Date(timestamp)
+
+  const date = dateObject.getDate()
+  const month = String(dateObject.getMonth() + 1).padStart(2, '0')
+  const year = dateObject.getFullYear()
+  const hour = dateObject.getHours()
+  const minute = dateObject.getMinutes()
+  const second = dateObject.getSeconds()
+  const milliSecond = dateObject.getMilliseconds()
+
+  const time = `${year}-${month}-${date} ${hour}:${minute}:${second}.${milliSecond}`
```

Comment:
I think the timezone could be useful here

Reply:
What about changing it to this then?
```
Z YYYY-MM-dd HH:mm:ss.SSS - <level> - <method> <route path> - <message>
```

It will be like this
```
GMT+0800 2017-02-14 20:51:48.000 - info - request completed
```

Reply:
Sorry, I mean the timezone offset:

```
2017-02-14 20:51:48.000+08:00 - info - request completed
```

This format should be compliant to the ISO 8601 
https://www.w3.org/TR/NOTE-datetime